Tests fail to import on macos 10.15, so you might just like to mark it broken for that here & now:

```
__________________ ERROR collecting tests/other/test_fork.py ___________________
ImportError while importing test module '/private/tmp/nix-build-python3.9-manticore-0.3.7.drv-0/source/tests/other/test_fork.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/lfayq1vz3szk4ji0s53v0x1ijcdn3qnx-python3-3.9.12/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/other/test_fork.py:3: in <module>
    from manticore.native import Manticore
manticore/native/__init__.py:3: in <module>
    install_helper.ensure_native_deps()
manticore/utils/install_helper.py:8: in ensure_native_deps
    raise ImportError(
E   ImportError: Missing some packages for native binary analysis. Please install them with pip3 install manticore[native].
------------------------------- Captured stdout --------------------------------
2022-05-23 19:52:56,918: [4886] m.c.manticore:WARNING: Manticore is only supported on Linux. Proceed at your own risk!
=============================== warnings summary ===============================
../../../../nix/store/srdbc3mvjr89y2gw7hf3jxmm07naajmr-python3.9-wasm-1.2/lib/python3.9/site-packages/wasm/types.py:264
  /nix/store/srdbc3mvjr89y2gw7hf3jxmm07naajmr-python3.9-wasm-1.2/lib/python3.9/site-packages/wasm/types.py:264: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
    isinstance(cur_field, collections.Callable) or

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR tests/other/test_fork.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
================== 13 deselected, 1 warning, 1 error in 1.42s ==================
```

--- a/pkgs/development/python-modules/manticore/default.nix
+++ b/pkgs/development/python-modules/manticore/default.nix
@@ -130,5 +130,7 @@ buildPythonPackage rec {
     license = licenses.agpl3Only;
     platforms = platforms.unix;
     maintainers = with maintainers; [ arturcygan ];
+    # m.c.manticore:WARNING: Manticore is only supported on Linux. Proceed at your own risk!
+    broken = stdenv.isDarwin;
   };
 }

